src/core_ocean/Registry.xml

```diff
@@ -190,80 +190,6 @@
 					possible_values="'split_explicit', 'RK4', 'unsplit_explicit'"
 		/>
 	</nml_record>
-	<nml_record name="ALE_vertical_grid" mode="forward">
```

Comment:
Why is this option disappearing?  We need this for a variety of coastal applications.